tests/e2e/tests/regression/cli/cli-command-helper.e2e.ts

```diff
@@ -107,19 +106,16 @@ test.skip
         //Click on Read More link for selected command
         await t.click(cliPage.readMoreButton);
         //Check new opened window page with the correct URL
-        await t.expect(getPageUrl()).contains(externalPageLink);
-        //Check that command info is displayed on the page
-        await t.expect(cliPage.cliReadMoreJSONCommandDocumentation().textContent).contains('JSON.SET');
+        await t.expect(getPageUrl()).eql(externalPageLink);
```

Comment:
The error message is missed. Please go over the whole file

Reply:
added